sebs/fission/fissionFunction.py

```diff
@@ -0,0 +1,40 @@
+from sebs.faas.function import Function, ExecutionResult
+import json
+import datetime
+import requests
+import logging
+
+
+class FissionFunction(Function):
+    def __init__(self, name: str):
+        super().__init__(name)
+
+    def sync_invoke(self, payload: dict):
+        url = "http://localhost:5051/benchmark"
```

Comment:
Ports should not be hardcoded - we either invoke functions as LibraryTrigger (`fission fn test`?) or HTTP trigger - we have a class for that, we just need the URL to make it work.

Reply:
as of  now I have implemented Http trigger

Reply:
I will change the hardcoded URL